libs/dkg/src/dkg.cpp

```diff
@@ -32,15 +33,14 @@ bn::G2                DistributedKeyGeneration::group_g_;
 bn::G2                DistributedKeyGeneration::group_h_;
 
 DistributedKeyGeneration::DistributedKeyGeneration(
-    MuddleAddress address, CabinetMembers const &cabinet, uint32_t const &threshold,
-    std::function<void(DKGEnvelope const &)> broadcast_function,
+    MuddleAddress address, std::function<void(DKGEnvelope const &)> broadcast_function,
     std::function<void(MuddleAddress const &, std::pair<std::string, std::string> const &)>
         rpc_function)
-  : cabinet_{cabinet}
-  , threshold_{threshold}
-  , address_{std::move(address)}
+  : address_{std::move(address)}
   , broadcast_function_{std::move(broadcast_function)}
   , rpc_function_{std::move(rpc_function)}
+  , dkg_state_gauge_{telemetry::Registry::Instance().CreateGauge<uint8_t>("dkg_state_gauge",
```

Comment:
Not sure if still relevant but can we have all metrics start with "ledger_" this is just makes Chris' life easier when coordinating these metrics at a network level

Reply:
Sure thing.